src/components/data/data-utils.js

```diff
@@ -131,25 +131,26 @@ export const ReferenceDataImporter = ({ onSuccess, onDismiss, namespace, name })
 export const ReferenceDataDeleter = ({ onSuccess, onDismiss, namespace, name, referenceDataType }) => {
   const [deleting, setDeleting] = useState(false)
 
-  return h(Modal, {
-    onDismiss,
-    title: 'Confirm Delete',
-    okButton: h(ButtonPrimary, {
-      disabled: deleting,
-      onClick: async () => {
-        setDeleting(true)
-        try {
-          await Ajax().Workspaces.workspace(namespace, name).deleteAttributes(
-            _.map(key => `referenceData_${referenceDataType}_${key}`, _.keys(ReferenceData[referenceDataType]))
-          )
-          onSuccess()
-        } catch (error) {
-          await reportError('Error deleting reference data', error)
-          onDismiss()
-        }
+  return h(DeleteConfirmationModal, {
+    objectType: 'reference',
+    objectName: referenceDataType,
+    onConfirm: async () => {
+      setDeleting(true)
+      try {
+        await Ajax().Workspaces.workspace(namespace, name).deleteAttributes(
+          _.map(key => `referenceData_${referenceDataType}_${key}`, _.keys(ReferenceData[referenceDataType]))
+        )
+        onSuccess()
+      } catch (error) {
+        reportError('Error deleting reference data', error)
+        onDismiss()
       }
-    }, ['Delete'])
-  }, [`Are you sure you want to delete ${referenceDataType}?`])
+    },
+    onDismiss
+  }, [
+    div(['Are you sure you want to delete the ', span({ style: { fontWeight: 600 } }, [referenceDataType]), ' reference data?']),
```

Comment:
I am not sure how to get to the reference deletion, but how would you feel about:
```suggestion
    div([`Are you sure you want to delete the ${b([referenceDataType])} reference data?`]),
```

Reply:
If you have an entity type `thing` for example, then you can upload a TSV that looks like:
```
membership:thing_set_id	thing
set1_name	thing1_name
set1_name	thing2_name
```

Then when you delete thing 1 or thing 2, it will prompt you to delete the set as well.

Reply:
Re: the `b` tag, sounds good to me. I'll go ahead and merge this as is and update #2962 to change that everywhere.

Reply:
Never mind https://github.com/DataBiosphere/terra-ui/pull/2951#discussion_r851443652, I got mixed up with something else. To delete reference data, first add some from the sidebar of the data tab.

![Screen Shot 2022-04-15 at 3 18 53 PM](https://user-images.githubusercontent.com/1156625/163623064-ef6dc7c4-e32b-4400-bf7a-22d2cfdeba09.png)

![Screen Shot 2022-04-15 at 3 19 01 PM](https://user-images.githubusercontent.com/1156625/163623067-c133edd1-224e-4016-995c-fd36d3ca2832.png)